Library/Formula/subversion.rb

```diff
@@ -156,6 +151,8 @@ def install
     if MacOS::CLT.installed?
       args << "--with-apr=/usr"
       args << "--with-apr-util=/usr"
+      # Ignore authz API bug in system httpd (fixed in httpd 2.4.16)
+      args << "--enable-broken-httpd-auth" if MacOS.version < :el_capitan
```

Comment:
I'm likely to sit on this for a few days. Apple have just released updates to Yosemite & Mavericks which remove the need to pass `--enable-broken-httpd-auth`, which means we could limit it to Mountain Lion and below which is nicer. Have asked the other maintainers if anyone objects.